tests/functional/s3cmd/tests.js

```diff
@@ -775,3 +777,21 @@ describe('s3cmd recursive delete with objects put by MPU', () => {
         deleteFile(upload16MB, done);
     });
 });
+
+describeSkipIfOldConfig('If no location is sent with the request', () => {
+    beforeEach(done => {
+        exec(['mb', `s3://${bucket}`], done);
+    });
+    afterEach(done => {
+        exec(['rb', `s3://${bucket}`], done);
+    });
+    // WARNING: change "file" to another locationConstrain depending
+    // on the restEndpoints (./config.json)
+    it('endpoint should be used to determine the locationConstrain', done => {
+        checkRawOutput(['info', `s3://${bucket}`], 'Location', 'file',
```

Comment:
you have to test this another way.  Electra's pr is going to break our ability to use 'info' on s3cmd until we implement more api calls that are made under the hood in an info call by s3cmd.  You can use Electra's new raw node testing suite.